openedx/jobs/openedxReleaseJob.groovy

```diff
@@ -0,0 +1,35 @@
+package openedx
+
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+
+// This is the job DSL responsible for creating the main pipeline job.
+pipelineJob('openedx-release-ci') {
+
+    definition {
+
+        parameters {
+            stringParam('OPENEDX_RELEASE_NAME', 'test-release',
+                        'open-release/${OPENEDX_RELEASE_NAME}')
+            choiceParam('DELETE_OR_KEEP', ['delete', 'keep'],
+                        'What should we do with the new release branches')
+
+        }
+
+        logRotator JENKINS_PUBLIC_LOG_ROTATOR(7)
+
+        cpsScm {
+            scm {
+                git {
+                    remote {
+                        credentials('jenkins-worker')
+                        github('edx/repo-tools', 'ssh', 'github.com')
+                        refspec('+refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/*')
+                        branch('master')
```

Comment:
Shouldn't `branch` be under `git` instead of `remote`?  And if we're only going to ever use master for this, can't we use a narrower `refspec` to speed up checkouts?